JSDemos/Demos/TreeList/ColumnChooser/description.md

```diff
@@ -1 +1,13 @@
-The Column Chooser is&nbsp;an&nbsp;element that allows a&nbsp;user to&nbsp;alter the set of&nbsp;visible columns at&nbsp;runtime. To&nbsp;open the column chooser, click the _&laquo;Column Chooser&raquo;_ button at&nbsp;the top of&nbsp;TreeList. Now, you can drag and drop column headers to&nbsp;and from the TreeList, thus showing or&nbsp;hiding columns. Use the column chooser&rsquo;s search box to&nbsp;quickly find the required column.
\ No newline at end of file
+The column chooser allows users to hide and display columns at runtime. To enable it, assign **true** to the **columnChooser**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#enabled) property. 
+
+To open the column chooser, users should click the button in the toolbar above the TreeList. The way users show and hide columns depends on the **columnChooser**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#mode):
+
+* *"dragAndDrop"*              
+Users drag and drop column headers to and from the column chooser.
+
+* *"select"*         
+Users select and unselect checkboxes with column names.
+
+If the [allowSearch](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#allowSearch) property is set to **true**, users can search through column names. In this demo, use the checkbox below the TreeList to toggle this feature.
+
+You can also hide specific columns programmatically. For this, set the **columns[]**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#visible) property to **false**. If a column should never be hidden, assign **false** to the **columns[]**.[allowHiding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowHiding) property. 
```

Comment:
I moved part of this to the top - looks to me that it makes more sense in the same paragraph with Column Chooser description. Looks to me like "how to change in code" and "how to not allow users to hide a column" are different thoughts - shouldn't really be in the same paragraph. 

```suggestion
To hide a column in code, set the **columns[]**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#visible) property to **false**. 
```